consensus/propagation/catchup.go

```diff
@@ -81,7 +81,7 @@ func (blockProp *Reactor) retryWants(currentHeight int64) {
 			}
 
 			// keep track of which requests we've made this attempt.
-			peer.AddRequests(height, round, missing)
+			peer.AddRequests(height, round, mc)
```

Comment:
why do we want to do this? it seems incorrect since we modify mc (on line 50) (the thing we are actually sending to the peer)

Reply:
wait nvm I switched up what should be here in this PR

Reply:
nvm, had it switched in my head... though we were replacing mc with missing :sweat_smile: